pyvista/core/utilities/reader.py

```diff
@@ -888,7 +888,9 @@ def _set_filename(self, filename) -> None:
 
     @property
     def number_time_points(self):  # noqa: D102
-        return self.reader.GetTimeSets().GetItem(self.active_time_set).GetSize()
+        if (item := self.reader.GetTimeSets().GetItem(self.active_time_set)) is None:
+            return 0
```

Comment:
This would be a breaking change as now anyone checking for `is None` will have code that breaks.  But I'm still in favor of this since it simplifies typing and expectation here.  If we merge this, we should implement for other readers to get a consistent behavior.